frontend/talentsearch/src/js/components/Router.tsx

```diff
@@ -588,18 +598,20 @@ export const Router: React.FC = () => {
 
   return (
     <>
-      <PageContainer
-        menuItems={menuItems}
-        authLinks={authLinks}
-        contentRoutes={[
-          ...talentRoutes(talentPaths),
-          ...authRoutes(authPaths),
-          ...profileRoutes(profilePaths, data?.me?.id),
-          ...(featureFlags.directIntake
-            ? directIntakeRoutes(directIntakePaths)
-            : []),
-        ]}
-      />
+      <Pending fetching={fetching} error={error}>
```

Comment:
If you hit this homepage and get a 401 (a bad token for example) you get have no way to log out since the menu is gone.

Reply:
Nice catch, hmm I don't have a quick fix for this right now so I'll remove it and make an issue.